File: leetcode/src/test/java/org/luvx/leetcode/java/medium/_2/SolutionTest.java
package org.luvx.leetcode.java.medium._2;

import org.junit.jupiter.api.Test;
import org.luvx.leetcode.link.ListNode;
import org.luvx.leetcode.util.LinkUtils;

public class SolutionTest {

    @Test
    public void addTwoNumbers() {
        final Solution exec = new Solution();
        ListNode l1 = LinkUtils.initLinked(7), l2 = LinkUtils.initLinked(5, 6, 4, 9);
        LinkUtils.printLink(exec.addTwoNumbers(l1, l2));
    }
}

File: leetcode/src/test/kotlin/org/luvx/leetcode/java/medium/_102/SolutionTest.kt
package org.luvx.leetcode.java.medium._102;

import org.luvx.leetcode.tree.TreeNode
import org.luvx.leetcode.out
import kotlin.test.BeforeTest
import kotlin.test.Test

class SolutionTest {
    private var root: TreeNode? = null

    @BeforeTest
    fun before() {
        root = TreeNode.of(
            5,
            TreeNode.of(
                4,
                TreeNode.of(
                    11,
                    TreeNode.of(7),
                    TreeNode.of(2)
                )
            ),
            TreeNode.of(
                8,
                TreeNode.of(
                    13
                ),
                TreeNode.of(
                    4,
                    null,
                    TreeNode.of(1)
                )
            )
        )
    }

    @Test
    fun levelOrder() {
        val exec = Solution()
        val levelOrder = exec.levelOrder(root)
        levelOrder.out()
    }
}